test/fuzz/BUILD

```diff
@@ -38,6 +38,23 @@ cc_binary(
     ],
 )
 
+cc_test(
+    name = "fuzz_doc_symbols",
+    srcs = [
+        "fuzz_doc_symbols.cc",
+    ],
+    tags = ["manual"],
+    visibility = ["//tools:__pkg__"],
+    deps = [
+        "//core",
+        "//main/lsp",
+        "//main/pipeline",
```

Comment:
We might not need this explicit dependency on pipeline, since we only directly interact with LSP.